README.md

```diff
@@ -1,5 +1,28 @@
-# Indy SDK
+# Indy SDK - Deprecated
 ![logo](https://raw.githubusercontent.com/hyperledger/indy-node/master/collateral/logos/indy-logo.png)
+
+## Deprecation Notice
+The Indy SDK has been deprecated in favor of the Indy and Aries shared libraries.  Please use the following libraries and projects to replace your use of the Indy SDK.
+
+### [indy-vdr](https://github.com/hyperledger/indy-vdr)
+A module for interacting with the [Indy Node](https://github.com/hyperledger/indy-node) ledger.
+
+### [aries-askar](https://github.com/hyperledger/aries-askar)
+A secure (encrypted at rest) storage and a key management service suitable for use with [Hyperledger Aries] agents and possibly other digital trust agents.  Askar is a replacement implementation (with lessons learned!) of the [indy-wallet] part of the [Hyperledger Indy SDK]. Askar has been demonstrated to be more performant and stable than the Indy SDK when under comparable load.
+
+### [indy-shared-rs](https://github.com/hyperledger/indy-shared-rs)
+Contains the following shared Rust libraries for Hyperledger Indy
+- `indy-credx`: Indy verifiable credential issuance and presentation (aka Anoncreds).
+- `indy-data-types`: Data type definitions for Schemas, Credential Definitions and other types related to credential issuance and processing.
```

Comment:
This should point to the AnonCreds RS repository I think. It supports what indy-shared-rs supports + ledger agnostic anoncreds.

https://github.com/hyperledger/anoncreds-rs

This is also where the work will be happening for making AnonCreds W3C data model compliant

Reply:
Especially if we are going to mark the repo as read-only, as @WadeBarnes and @ryjones were discussing, we should update the links as @TimoGlastra recommends.

Reply:
That's the sort of feedback I was looking for.  Thanks!